PendingReleaseNotes.md

```diff
@@ -16,7 +16,7 @@
   - The on-host log directory for OSDs was set incorrectly to `<dataDirHostPath>/<namespace>/log`;
     fix this to be `<dataDirHostPath>/log/<namespace>`, the same as other daemons.
   - Use the mon configuration database for directory-based OSDs, and do not generate a config
-
+- Ceph CSI now uses 4 different keys, divided for 'cephfs' and 'rbd' where each have a 'mounter' and 'provisioner' key with the right restrictions.
```

Comment:
As discussed, we can add a note like this to the release notes for the patch release, but no need to add it here since we'd have merge conflicts.